### ISSUE TITLE
Parachutes no workerino

### ISSUE THREAD

**Issue by LookMomImWalrus** [StarQuestMinecraft/StarQuestPublic#1128]
Here's what happened. Ask lone traveller, she can back this up.
1. I had some leather, and went for a fly in a parachute
2. As I hit the ground, I instantly die.
3. I ask mod for tp back, but it takes too long and my items despawn
4. Mod tries refund death, but it doesn't work.

Therefore:
-Parachutes are broken, you take fall damage on hitting ground
-Death refund is broken, doesn't work
-Wookie needs to be refunded his set of god gear and tools because it's not cool to lose those things to a bug

Thanks.

**Comment by LookMomImWalrus** [comment]
edit: My ign is wookiejesuz.

**Comment by gregolegomonkey** [comment]
If no one else was in the area, and where you died isn't near a planet's spawn/center, then they _should_ still be sitting there, at least for a little while.

**Comment by LookMomImWalrus** [comment]
No, lone traveller was afk in the chunks near it.

**Comment by mcdouble** [comment]
I've noticed that SOMETIMES you take fall damage and sometimes you don't

**Comment by awangatboo** [comment]
Yes, NickMiste wrote the parachute plugin and he said that he could never figure out why for some reason on rare occasions you can take fall damage.
